docs/insomnia/git-sync.md

```diff
@@ -77,4 +78,4 @@ Designer does not currently support the ability to resolve conflicts. If changes
 Here are some strategies to help with conflicts:
 
 * Each team member should make changes in a separate branch to avoid conflicts. Changes should be merged into master once reviewed and approved by other team members (eg. GitHub pull request).
-* If a conflict occurs on pull, delete the branch locally and re-fetch it from the branches dialog.
\ No newline at end of file
+* If a conflict occurs on pull, delete the branch locally and re-fetch it from the branches dialog.
```

Comment:
No actual change here 🤔 appears to be a line-ending issue (I created the commit from within github), markdownlint should be able to manage this when we add it.